README.md

```diff
@@ -112,6 +112,32 @@ https://testing.imgix.net/image.jpg?w=196 196w,
 https://testing.imgix.net/image.jpg?w=8192 8192w
 ```
 
+### Min and Max Width Ranges
```

Comment:
Somewhere in this section I feel like we need to mention the DPR gotcha.

> Remember that browser's take device pixel ratio into account when selecting which image to download from a `srcset`. For example, even if you know your image will render no larger than 1000px, specifying `max_width: 1000` will give your users with DPR higher than 1 no choice but to download and render a low-resolution version of the image.

Something like this?